src/_includes/layouts/homepage.html

```diff
@@ -6,35 +6,43 @@
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/styles.css" />
     <link rel="stylesheet" href="/css/navbar.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/homepage.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
     <title>Lunch.dev Community Calendar</title>
   </head>
   <body data-typeface="system-ui">
-	{% include partials/header.html %}
-    <main data-layout="centered-single-column">
-      <h1>Lunch.dev Community Calendar</h1>
+    {% include partials/header.html %}
 
-      <h2 id="upcoming-events-section">Upcoming Events</h2>
-      <ul aria-labelledby="upcoming-events-section">
+    <main data-layout="centered-single-column">
+      <h2>Upcoming Events</h2>
+      <section id="upcoming-events-section" class="events-section">
         {% for event in collections.upcomingEvents %} {{ event | log }}
-        <li>
-          <a href="{{ event.url }}"> {{ event.data.title }} </a>
+        <article class="event-item">
+          <h3>{{ event.data.title }}</h3>
           {% if event.data.type %}
-          <span>{{ event.data.type }}</span>
+          <div>{{ event.data.type }}</div>
           {% endif %}
-        </li>
+          <a href="{{ event.url }}">View Details &#8250;</a>
```

Comment:
Some screenreader users navigate by skipping from link to link. In that case, though, they'd hear "View Details, link. View Details, link. View Details, link." and so forth on our page. While sighted users know those links are different, because the links are close to the event titles, blind users who use screenreaders don't necessarily get that context in all navigation modes. Because of that, [it's best to make the link announcement distinct for each link.](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

I recommend using `aria-label` here, so that we can curate the screenreader experience without necessarily changing the visual experience. Something like:

```html
<a href="{{ event.url }}" aria-label="View Details for {{ event.data.title }}">View Details &#8250;</a>
```

What do you think?